### PR TITLE
Add image selection for minecraft java eggs for java 8 & 11

### PR DESCRIPTION
**Pull request by TekExplorer**
Set minecraft eggs to have both java versions as options by default.
is there some way to set a default or an alias for images so that users know what they are?

### CODE REVIEW

**Comment by TekExplorer** [comment]
actually, perhaps the deprecated `image` could be used to set the default?

**Comment by matthewpi** [comment]
> actually, perhaps the deprecated `image` could be used to set the default?

No point as these changes will only affect newer versions of the panel that have support for multiple images.

**Comment by TekExplorer** [comment]
> > actually, perhaps the deprecated `image` could be used to set the default?
> 
> No point as these changes will only affect newer versions of the panel that have support for multiple images.

what i mean is, while `images` can be used to specify the list, `image` could be used to set the default image *for* the multi-image supported versions, which plays well with keeping backwards compatibility i think as older versions would just use that "default"

**Comment by DaneEveritt** [comment]
The default image is just the first one, no need for complexity.

File: database/seeds/eggs/minecraft/egg-sponge--sponge-vanilla.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "author": "support@pterodactyl.io",
     "description": "SpongeVanilla is the SpongeAPI implementation for Vanilla Minecraft.",
     "features": ["eula"],
-    "image": "quay.io\/pterodactyl\/core:java-glibc",
+    "images": "quay.io\/pterodactyl\/core:java-glibc",
```
Comment (DaneEveritt) [inline database/seeds/eggs/minecraft/egg-sponge--sponge-vanilla.json]: This needs to be an array.

**Review by DaneEveritt — CHANGES_REQUESTED** [review]
see comment

File: database/seeds/eggs/minecraft/egg-sponge--sponge-vanilla.json (hunk 1 above)
Comment (TekExplorer) [reply database/seeds/eggs/minecraft/egg-sponge--sponge-vanilla.json]: good catch, didnt notice

**Pull request merged**